package/utils/apk/Makefile

```diff
@@ -0,0 +1,54 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=apk
+PKG_VERSION:=9f07a3447ea1e8fb67cdbd5c30b2ea144e826490
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=apk-tools-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://gitlab.alpinelinux.org/alpine/apk-tools/-/archive/$(PKG_VERSION)
+PKG_HASH:=59c3ee8d82e09f6d18f90db90deea70c82651ab63f2eb58b931321c9114ed98e
+PKG_BUILD_DIR:=$(BUILD_DIR)/apk-tools-$(PKG_VERSION)
+HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/apk-tools-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=LICENSE
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/meson.mk
+
+define Package/apk
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=apk package manager
+  DEPENDS:=+zlib +libopenssl +libpthread @!arc
+  URL:=$(PKG_SOURCE_URL)
+endef
+
+MESON_HOST_ARGS += \
+	-Ddocs=disabled
+
+MESON_ARGS += \
```

Comment:
why get rid of -Dlua=disabled ?

Reply:
To have --help

Reply:
--help requires libluaapk.so AFAIK

Reply:
Lua is only required to build the help database.  It can also be pre-built.

Reply:
Thak you, updated and it works.